src/user/announcements.rst

```diff
@@ -8,6 +8,16 @@ Our announcements are published to an RSS feed `here <https://conda-forge.org/do
 2022
 ----
 
+2022-08-24: Dropping Python 3.7
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Conda-Forge has been providing support for Python 3.7 for 4 years now.
```

Comment:
For reference, the 4 years is drawn from PR ( https://github.com/conda-forge/python-feedstock/pull/183 ), which added Python 3.7.0 to conda-forge on July 13, 2018.